### ISSUE TITLE
Invisible for Quick Control widget of Source Localization in MNE-Scan

### ISSUE THREAD

**Issue by BiboyLin** [mne-tools/mne-cpp#709]
![image](https://user-images.githubusercontent.com/34125254/96700190-c20a4180-13c1-11eb-9fa8-a612751adb37.png)
Both in dynamic or Static Version of MNE-Scan, there is no Quick Control widget of Source Localization as the screenshot displayed. 
MNE-CPP version: Dynamic and Static Version(0.1.6) downloaded in 2020/10/21
OS: Windows 10

**Comment by BiboyLin** [comment]
Also, it is hard to follow the homepage guidance(Learn/MNE Scan/Real-time Source Localization, https://mne-cpp.github.io/pages/learn/scan_sourceloc.html) to use this function. Did **MNE** module the page mentioned changed to **Real-time Source Localization** in the current version? I will be very appreciated if you can update the guides. Thanks a lot!

**Comment by RDoerfel** [comment]
Hi @BiboyLin, in order to see the plugin, you have to add the [Forward Solution Plugin](https://mne-cpp.github.io/pages/learn/scan_forward.html) as well. The documentation should cover pipelines for source localization as well.

**Comment by LorenzE** [comment]
We updated the documentation. Let me know if it works for you know. Feel free to reopen if you run into problems.
